.gitignore

```diff
@@ -1,2 +1,133 @@
-__pycache__
+# Created by https://www.gitignore.io/api/python
+# Edit at https://www.gitignore.io/?templates=python
```

Comment:
Can you drop the .gitignore change from this PR? A significant amount of this is unnecessary.